projects/gpsd/project.yaml

```diff
@@ -0,0 +1,16 @@
+homepage: "https://gpsd.io"
+language: c
+primary_contact: "gpsd-dev@lists.nongnu.org"
```

Comment:
This is not good. Members of the list will get emails but wont be able to access bugs.

Reply:
I understand I'm still trying to contact the GPSD team. 
If I get a member mail ID, who can fix bugs, I will change it. 
Until then, hold it.